src/components/AudioTrack/AudioThumb.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <!-- Should have a fixed width -->
```

Comment:
```suggestion
  <!-- Should have a fixed width -->
```

I'm not 100% sure what this comment means. It reads like a `@TODO` but I think it's meant to be more of an suggestion for implementing the audio thumb? I'm not certain.

Reply:
It's an instruction for the users of this component. I should probably add an explanation for this in the Storybook for the component.

Reply:
thanks!